index.html

```diff
@@ -31,33 +32,42 @@
             <a class="nav-link active" aria-current="page" href="#">Home</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="#">Link</a>
+            <a class="nav-link" href="#">About us</a>
           </li>
-          <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown"
-              aria-expanded="false">
-              Dropdown
-            </a>
-            <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
-              <li><a class="dropdown-item" href="#">Action</a></li>
-              <li><a class="dropdown-item" href="#">Another action</a></li>
-              <li>
-                <hr class="dropdown-divider">
-              </li>
-              <li><a class="dropdown-item" href="#">Something else here</a></li>
-            </ul>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Our customers</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link disabled">Disabled</a>
+            <a class="nav-link" href="#">Contact us</a>
           </li>
         </ul>
-        <form class="d-flex m-0">
-          <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-          <button class="btn btn-outline-success" type="submit">Search</button>
-        </form>
+        <div class="d-flex">
+          <button class="btn btn-outline-primary">Log in</button>
+        </div>
       </div>
     </div>
   </nav>
+  <!-- Main Page Section -->
+  <section id="main-page" class="d-flex flex-column flex-md-row my-4 align-items-center justify-content-center">
+    <div class="w-full md:w-1/2 mb-4 mb-md-0 me-md-4 ">
+      <span class="fs-1 fw-bolder text-center d-block">Your best recruitment agency</span>
+      <div class="d-flex flex-column mt-4 align-items-center">
+        <div class="fs-4 fw px-2 py-1 mb-3 text-center">
+          Look for your perfect development job
+        </div>
+        <div class="fs-3 fw-bold px-2 py-1 mb-3 text-center">
+          or
+        </div>
+        <div class="fs-4 fw px-2 py-1 mb-3 text-center">
+          Publish your job offers and find the best software professionals
+        </div>
+        <button class="btn btn-primary w-25">Join now for free</button>
```

Comment:
This button looks to small on small screens. Just remove the custom width, looks okay without it.

```suggestion
        <button class="btn btn-primary">Join now for free</button>
```